### PR TITLE
Back navigation issue

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/features/notification/types.ts [hunk 1]
```diff
@@ -99,6 +99,7 @@ export interface NotificationContent {
   appId?: string
   userId?: string
   AppName?: string
+  OfferName?: string
```
Comment (oyo) [inline cx-portal/src/features/notification/types.ts:102]: starting variables or props with upper case is not common practice in Typescript - please convert to `offerName` . Also the one above

File: cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx [hunk 2]
```diff
@@ -257,7 +258,10 @@ export default function NotificationItem({
               }}
             >
               {' '}
-              {t(`${item.typeId}.title`)}
+              {t(`${item.typeId}.title`, {
+                app: item.contentParsed?.AppName,
+                offer: item.contentParsed?.OfferName,
```
Comment (oyo) [inline cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx:263]: naming

**Review by oyo — CHANGES_REQUESTED** [review]
tiny change regarding naming conventions

File: cx-portal/src/features/notification/types.ts (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/features/notification/types.ts:102]: These two are from api response.
`content
: 
"{\"OfferId\":\"34e9dabb-ad5e-42c5-b6e0-4a4d7af4e391\",\"CompanyName\":\"CX-Test-Access\",\"OfferName\":\"Some test app1\"}"`

File: cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx (hunk 2 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/NotificationItem.tsx:263]: same as mentioned above

**Review by oyo — APPROVED** [review]
okay, now approved

**Pull request merged**